include/smack/SmackRep.h

```diff
@@ -83,7 +89,16 @@ class SmackRep {
   Program& getProgram() { return program; }
 
 private:
-  void addInit(unsigned region, const Expr* addr, const llvm::Constant* val);
+  // 
+  // Pointer size: 32
+  //
+  static const unsigned ptrsize;
+  //
+  // A flag for bit-vector
+  //
+  static bool BIT_VECTOR;
```

Comment:
I understand. Let's finalize the use case. 

First, SmackOptions is invoked by a flag like -bit-vector when smack binary is executed if I remeber is correctly. Suppose we would like to switch to bit-vector following the rule that if manual selection enabled by a flag of scripts is turned on or if it is not while a detection pass says that it shoule be. The boolean expression for this rule is !manualSelection || !isBVRequired. Should I evluate this expression in the llvm2bpl script and invoke the SmackOptions::BitVector?